blog/cr1-for-mars.adoc

```diff
@@ -0,0 +1,118 @@
+= CR1 for Eclipse Mars - more Docker, OpenShift and WildFly 10
+:page-layout: blog
+:page-author: akazakov
+:page-tags: [release, jbosstools, devstudio, jbosscentral]
+:page-date: 2015-09-22
+
+We are happy to announce JBoss Tools 4.3 CR1 and Red Hat JBoss Developer Studio 9 CR1 for Eclipse Mars are now available.
+
+image::/blog/images/jbosstools-jbdevstudio-blog-header.png[]
+
+== What is new ? 
+
+The full details of what is new is available on http://tools.jboss.org/documentation/whatsnew/jbosstools/4.3.0.CR1.html[this page]. Some highlights are below.
+
+== Eclipse Mars.1
+
+This version of JBoss Tools targets Eclipse Mars.1 which includes many bug fixes and some improvements.
+
+=== WildFly 10 and EAP 7 Server Adapters
+
+New server adapters for JBoss EAP 7 and WildFly 10 have been added to the toolset, allowing you to enjoy all the past benefits, but with all the newest runtimes.
+
+image::images/cr1-for-mars/wildfly10.png[width="500"]
+
+=== Docker tooling
+
```

Comment:
Add a segment like:

Note: The Docker tooling is part of Eclipse Mars.1 We worked on these improvements to make them rock in JBoss Tools with OpenShift support - so we wanted to highlight them.